es6/use-class-syntax-to-define-a-constructor-function.js

```diff
@@ -0,0 +1,16 @@
+// Only change code below this line
+class Vegetable {
+  constructor(carrot) {
```

Comment:
please change this
The example uses carrot (cenoura in portuguese) to be a vegetable name.
it's not to name the parameter as carrot
Think a little and put a name that make sense. Rememnber that the constructor receive the parameter and create an attribute name to keep the value.